src/OpenLoco/Windows/CompanyList.cpp

```diff
@@ -576,7 +574,7 @@ namespace OpenLoco::Ui::Windows::CompanyList
 
         if (window != nullptr)
         {
-            if (Input::isToolActive(_toolWindowType, _toolWindowNumber))
+            if (Input::isToolActive(WindowManager::getToolWindowType(), WindowManager::getToolWindowNumber()))
```

Comment:
This is an odd `if` its basically saying is any tool active and if it is cancel it then bring to front. It would make more sense to just call toolCancel and then do bringToFront.